KinSDK/KinSDK/KeyStore.swift

```diff
@@ -34,18 +34,16 @@ public class StellarAccount: Account {
         return try accountData().extra
     }
 
-    public var sign: (([UInt8]) throws -> [UInt8])?
-
-    public func sign(message: [UInt8], passphrase: String) throws -> [UInt8] {
-        guard let signingKey = secretKey(passphrase: passphrase) else {
+    public func sign(message: [UInt8]) throws -> [UInt8] {
+        guard let signingKey = secretKey() else {
             throw KeyStoreErrors.noSecretKey
         }
 
         return try KeyUtils.sign(message: message, signingKey: signingKey)
     }
 
-    fileprivate func secretKey(passphrase: String) -> [UInt8]? {
```

Comment:
@yosriz be aware of changes like this, removing the passphrase externally, in order to move the ios sdk closer to androids